### ISSUE TITLE
Installation of aquatic-animal-excretion is failing

### ISSUE THREAD

**Issue by apoorvaeternity** [weecology/retriever#1132]
![aqua](https://user-images.githubusercontent.com/21103831/38459565-d0d994d8-3ac8-11e8-974c-d7f48ad82937.png)
The url http://onlinelibrary.wiley.com/store/10.1002/ecy.1792/asset/supinfo/ecy1792-sup-0001-DataS1.zip?v=1&s=3a9094a807bbc2d03ba43045d2b72782bfb348ef is resulting into a 404 page.
![screenshot from 2018-04-08 01-06-57](https://user-images.githubusercontent.com/21103831/38459594-363aa3b2-3ac9-11e8-9f09-44ec81518a6e.png)

**Comment by ethanwhite** [comment]
Here's the new link:
http://onlinelibrary.wiley.com/store/10.1002/ecy.1792/asset/supinfo/ecy1792-sup-0001-DataS1.zip?v=1&s=3a9094a807bbc2d03ba43045d2b72782bfb348ef

Unfortunately it looks like it has some of the redirect issues so the new url won't fix this until we implement #1114.

**Comment by zhangcandrew** [comment]
This seems to be have been solved after #1114 and the new link. Feel free to re-open if it becomes an issue again.
